abilities/chinese/great-wall-buff-chi-4.json

```diff
@@ -0,0 +1,44 @@
+{
+  "id": "great-wall-buff-chi-4",
+  "baseId": "great-wall-buff-chi",
+  "type": "ability",
+  "name": "The Long Wall",
+  "pbgid": 2076747,
+  "attribName": "great_wall_buff_chi",
+  "age": 4,
+  "civs": [
+    "ch"
+  ],
+  "description": "All units standing on Walls gain +25% ranged damage.",
+  "classes": [],
+  "displayClasses": "",
+  "costs": {
+    "gold": 0,
+    "wood": 0,
+    "food": 0,
+    "stone": 0,
+    "total": 0,
+    "time": 0,
+    "popcap": 0
+  },
+  "producedBy": [],
+  "icon": "https://data.aoe4world.com/images/abilities/great-wall-buff-chi-1.png",
+  "active": "always",
+  "auraRange": 0,
+  "effects": [
+    {
+      "property": "rangedAttack",
+      "select": {
+        "id": [
```

Comment:
Could also go for `class: [['ranged','infantry']]` so if at some point Chinese units would get added they are also targeted